observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+The default platform monitoring stack is available for use after installing {product-title}.
+The in-cluster stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that routes alerts.
+
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+Depending on which users will use the monitoring stack and for what purposes, as a cluster administrator, you can configure monitoring components to suit the needs of different users in various scenarios.
+
+[id="core-platform-monitoring-first-steps_{context}"]
+== Core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
+These activities include configuring options for Prometheus and Alertmanager, setting up short-term and long-term storage, and adding service monitors.
```

Comment:
except Prometheus and Alertmanager, we can also configure  other components
as for `adding service monitors`, I think  it there is something wrong with the added service monitors by users, Red Hat does not have the duty to fix it. need developer help to confirm

Reply:
For the Core platform side of things, ServiceMonitors are out of scope. I would just drop this part here since we don't support ServiceMonitor additions by [admin] users.

I think e can also drop `short-term [storage]`. Out of the box Prometheus will store data in the container file system. When the pod is restarted that data is lost. I assume that is what short-term storage means here? If so there is no setup required.

Reply:
removed the service monitor bit and just said `setting up storage`